### PR TITLE
updated dropshare (4-3802)

### PR DESCRIPTION
**Pull request by elbracht**

### CODE REVIEW

File: Casks/dropshare.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'dropshare' do
-  version '3.13.1'
-  sha256 'e695a456c39c33085e1f69f169e166fe8027a6a22350a445a016cbacee317e73'
+  version '4-3802'
+  sha256 '8b8c049eb0dffd938fce9c80dfd95d227e1fbfab7f060737e7dce676cade2f47'
 
   # d2wvuuix8c9e48.cloudfront.net was verified as official when first introduced to the cask
-  url 'https://d2wvuuix8c9e48.cloudfront.net/Dropshare-latest.zip'
+  url 'https://d2wvuuix8c9e48.cloudfront.net/Dropshare4-3802.app.zip'
```
Comment (victorpopkov) [inline Casks/dropshare.rb]: You can use string interpolation here to reuse the version number:
```ruby
  url "https://d2wvuuix8c9e48.cloudfront.net/Dropshare#{version}.app.zip"
```

File: Casks/dropshare.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   appcast 'https://getdropsha.re/sparkle/Dropshare.xml',
```
Comment (victorpopkov) [inline Casks/dropshare.rb]: The appcast also needs to be changed to the one used in Dropshare 4 with the new checkpoint:
```ruby
  appcast 'https://getdropsha.re/sparkle/Dropshare4.xml',
          checkpoint: 'd87e77caf46666242b66bbef7becf16409cf0c585446510b3988765016e07477'
```

**Comment by skaren** [comment]
Can I use `cask-repair` again to update the issues?

**Comment by victorpopkov** [comment]
> Can I use `cask-repair` again to update the issues?

Nope, I guess. There is no `--force` flag present in `git push` commands anywhere in the [cask-repair](https://github.com/vitorgalvao/tiny-scripts/blob/master/cask-repair) script. I'm not sure, though, but it's better to make fixes manually when commit was already done by the script.

**Comment by victorpopkov** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
> Can I use `cask-repair` again to update the issues?

No, but you can close and resubmit if that’s easier for you.

**Comment by skaren** [comment]
I squash the two commits.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 1d2aaed2bf9ba1da00e9945c756de3165216701b to keep commit history cleaner. Your contribution is still credited to you.

**Comment by skaren** [comment]
Now I see there is one more issue with the cask.

```
Error: It seems the symlink source is not there: '/opt/homebrew-cask/Caskroom/dropshare/4-3802/Dropshare.app'
```

The problem is the app name. This should be `app 'Dropshare 4.app'` instead of `app 'Dropshare.app'`.

What is the best way to resolve this? Should I create a new pull request or use this?

**Comment by victorpopkov** [comment]
You should create a new PR.

**Pull request closed without merging**